pkgs/development/python-modules/glean-parser/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, pkgs
+, python3Packages
+, setuptools-scm
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "glean_parser";
+  version = "4.3.1";
+
+  disabled = python3Packages.pythonOlder "3.6";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-wZSro1pX/50TlSfFMh71JlmXlJlONVutTDFL06tkw+s=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    appdirs
+    click
+    diskcache
+    jinja2
+    jsonschema
+    pytest
```

Comment:
```suggestion
```
This makes overriding pytest versions harder than necessary. Also are you sure some of those are not checkInputs?

Reply:
Good point, moving the testing stuff to `checkInputs` now